docs/generate_db_schema

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
Note that [Fedora's Python packaging guidelines](https://fedoraproject.org/wiki/Packaging:Python#Multiple_Python_Runtimes) will require this to be changed to an explicit Python version (and not to use env too), like ```/usr/bin/python3```. It is possible to use ```sed``` in the specfile to accomplish this, but I recommend using an explicit shebang here instead.

Reply:
Anitya isn't yet packaged in Fedora, but thanks for pointing at this.

Reply:
Tried with `/usr/bin/python3` and this isn't working. Probably because of the `sqlalchemy_schemadisplay` is python2 import.
However it's working with `/usr/bin/env python3`.